lib/kitchen/directions/bake_unnumbered_tables.rb

```diff
@@ -4,7 +4,7 @@ module Kitchen
   module Directions
     module BakeUnnumberedTables
       def self.v1(book:)
-        book.tables('$.unnumbered').each do |table|
+        book.tables(only: :unnumbered_table_to_bake?).each do |table|
```

Comment:
question --

Reply:
When they do, they're getting styles like regular tables (being centered, proper paddings). I assumed that this is what we want, but maybe let's confirm this with @lwtchu.

Reply:
They have their own styling that Jenn implemented a while back. We should keep how they look like in the screenshot but just without table numbering

Reply:
i think that's correct (only `td` styles, no other table styles). as far as i'm aware, the hljs line number tables aren't tables -- the line numbering package found it useful to leverage the structure of an html table in order to put numbers on the lines, but otherwise, these elements should appear the same as a code snippet that doesn't have line numbers, eg. not in a table

Reply:
I tried to removed styles in baked.xhtml that are inherited from regular tables and this is what it looks like:

![image](https://github.com/openstax/cookbook/assets/40228854/785b5d47-ea3f-4fbd-9684-968deb33392f)

Code is left aligned and there are empty spaces between borders. Is this acceptable? If yes, then I can remove `hljs-ln` class from table baking.

Reply:
Yes :+1: This could be done by adding property to `<table>` tag, so I'll do this in shape that Jenn created. For this we don't need to bake hljs table, as a table, so let me do a quick fix for this PR.